internal/fixtures/policyAsCode/custom-keys/custom-rule.yaml

```diff
@@ -0,0 +1,13 @@
+properties:
```

Comment:
the file name is a bit confusing. this is not a custom-rule, this is a yaml schema